tests/NHSD.BuyingCatalogue.API.IntegrationTests/NHSD.BuyingCatalogue.API.IntegrationTests.csproj

```diff
@@ -22,6 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\src\NHSD.BuyingCatalogue.Solutions.Contracts\NHSD.BuyingCatalogue.Solutions.Contracts.csproj" />
```

Comment:
This is required because the `PublishedStatus` enum is used when setting up the test database with solutions. Without it we would have magic numbers in our background tables. This doesn't affect the integrity of the actual tests (scenarios) as they have no dependency on the enumeration.

The use of the enumeration means that we can have meaningful names in our background tables instead of magic numbers (e.g. 'Published' instead of 3). It also means the tests will always use valid values when populating the database (as these should never be invalid).

Reply:
Agreed we will have to be mindful in future PRs & want to be careful with the types used in any of the tests but to keep the data consistent when setting up the database this will be useful going forward.